README.md

```diff
@@ -12,14 +14,20 @@ Alternatively, you can get the path with `get_curator_path`.
 
 ## Building the package:
 
-Make sure you edit the hash in `testdir/test_basic.py`, `curatorbin/__init__.py` and `evergreen.yml`. 
-You can use `./changeover.sh ${OLD_HASH} ${NEW_HASH}` for this.
+You can use `./updateFromCurator.sh <major|minor|patch>` for this.
+
+For example, `./updateFromCurator.sh minor` will update the minor version, which would turn `1.2.3` to `1.3.0` in setup.py.
 
-Also, increment the version in the `setup.py` and `evergreen.yml` files. 
+This will be used to increment the version in the `setup.py` and `evergreen.yml` files. 
```

Comment:
I would put this implementation detail about how it updates the version in the script itself. The README documentation should stay high-level and general.